tests/Unit/ActionTesting.hpp

```diff
@@ -501,6 +501,14 @@ class MockArrayElementProxy {
     local_algorithm_.template simple_action<Action>();
   }
 
+  // This doesn't do anything, but it allows to test actions that set this flag.
+  bool terminate{false};
+  void set_terminate(bool t) noexcept { terminate = t; };
```

Comment:
It should forward the call to `set_terminate` to the correct thing (`local_algorithm_`) because that's what the actual code does that is being mocked.